### PR TITLE
Bump up kernel version, fw and ucode

### PR DESCRIPTION
**Pull request by rouming**
This just bumps up a kernel version in the `kernel-commits.mk`, `kernel-version.mk` and also increases fw and ucode to the latest available by today.

So these versions will be used:

amd64-generic: v6.1.106
amd64-rt: v6.1.105
arm64-generic: v6.1.106
riscv64-generic: v6.1.106

Update: this should be merged after this is merged: https://github.com/lf-edge/eve-kernel/pull/133

### CODE REVIEW

File: kernel-commits.mk [hunk 1]
```diff
@@ -2,8 +2,12 @@ KERNEL_COMMIT_amd64_v5.10.186_generic = d61682724485
 KERNEL_COMMIT_amd64_v6.1.38_generic = 8859f43ee8c9
 KERNEL_COMMIT_amd64_v6.1.38_rt = 37bc37278441
 KERNEL_COMMIT_amd64_v6.1.68_generic = 45e0ac394428
+KERNEL_COMMIT_amd64_v6.1.105_rt = 8fe1d58eff31
```
Comment (OhmSpectator) [inline kernel-commits.mk:5]: I'm just curious: why is 6.1.106 ok for generic but not for RT, which requires 6.1.105 =)

File: kernel-commits.mk [hunk 2]
```diff
@@ -2,8 +2,12 @@ KERNEL_COMMIT_amd64_v5.10.186_generic = d61682724485
...
+KERNEL_COMMIT_amd64_v6.1.106_generic = ee5e09825b81
 KERNEL_COMMIT_arm64_v5.10.104_nvidia = 7b9a55e0b659
 KERNEL_COMMIT_arm64_v5.10.192_nvidia = c2b4176f82b1
 KERNEL_COMMIT_arm64_v5.10.186_generic = 804663a82829
 KERNEL_COMMIT_arm64_v6.1.38_generic = acbfadebeb76
+KERNEL_COMMIT_arm64_v6.1.106_generic = ee5e09825b81
```
Comment (rucoder) [inline kernel-commits.mk:11]: same commit SHAs seem very suspecious

**Review by rucoder — CHANGES_REQUESTED** [review]

File: kernel-commits.mk (hunk 2 above)
Comment (rouming) [reply kernel-commits.mk:11]: arm not yet pushed, this is reflected in the description of the PR.

File: kernel-commits.mk (hunk 1 above)
Comment (rouming) [reply kernel-commits.mk]: there is no stable v6.1.106 for RT available

**Review by eriknordmark — APPROVED** [review]
LGTM but the commit message linter is not happy

**Review by eriknordmark — APPROVED** [review]
Once more

**Comment by christoph-zededa** [comment]
Can we please have commit messages that are relevant?
F.e.:
```
e61cce864cfa: Dockerfile: add pahole dependency
```
is not only irrelevant for this repository, but it is even mentioned twice.

**Comment by rouming** [comment]
> Can we please have commit messages that are relevant? F.e.:
> 
> ```
> e61cce864cfa: Dockerfile: add pahole dependency
> ```
> 
> is not only irrelevant for this repository, but it is even mentioned twice.

I understand your concern to minimize the log for autogenerated commit, which described the difference in eve-kernel. Here in this PR I manually minimized the log by reducing to 5 commits for each branch and put the link on github for those who are thrilled to see the full log. Christoph, I suggest you to raise this concern to @rucoder and contribute to the `tools/update_kernel_commits.py` with a PR, then it makes more sense to change things rather to complain in each commit which aims to update kernel SHA.

**Comment by OhmSpectator** [comment]
By the way, I want to fix the commit message linter to allow kernel update commits in the form they are done now. Ignore it until then.

**Review by eriknordmark — APPROVED** [review]
LGTM - ignoring the commit linter.

**Pull request merged**